android-agent/src/main/java/io/opentelemetry/android/internal/services/network/CarrierFinder.java

```diff
@@ -3,22 +3,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.android.instrumentation.network;
+package io.opentelemetry.android.internal.services.network;
 
 import android.os.Build;
 import android.telephony.TelephonyManager;
 import androidx.annotation.RequiresApi;
+import io.opentelemetry.android.internal.services.network.data.Carrier;
 
 @RequiresApi(api = Build.VERSION_CODES.P)
-class CarrierFinder {
+public class CarrierFinder {
```

Comment:
These classes that are still java and have moved to `internal` packages should get the boilerplate disclaimer about not being for public use and subject to change.

Reply:
Thanks, I've made the changes.